spec/serializers/api/v2/serializable_provider_spec.rb

```diff
@@ -11,4 +11,5 @@
   subject { resource.as_jsonapi.to_json }
 
   it { should be_json.with_content(type: 'providers') }
+  it { should be_json.with_content(attributes: { provider_code: provider.provider_code }) }
```

Comment:
beyond scope but can we add the `institution_name` to the spec as well please.

Also maybe it be wise to rename `institution_name` to `provider_name` as that will be the last mention of `institution_*`

Reply:
Yeah great idea, was planning on renaming that in a separate PR :)

Reply:
Feel like it's best to leave that until later, I'll put in a Trello card for now.

Reply:
Card: https://trello.com/c/U5X1SNps